docs/profiling.md

```diff
@@ -0,0 +1,108 @@
+# vHive profiling tool
+
+The tool focus on the performance characteristics of vHive framework and identify potential 
+performance bottleneck of colocating VMs on a single host. The motivation of this tool is to 
+improve the parallelism of serverless architecture and commercial benefits for cloud vendors.
+
+## Methodology
+
+The tool is for benchmarking vHive instances by recording hardware counters, requests per second
+ (RPS) and execution time for functions running inside instances.
+
+The tool includes the following components:
+- A loader function issues requests to vHive step by step until the RPS reaches the maximum. At 
+  each step, the function issues a percentage of the maximum RPS (e.g., 5% of the maximum RPS) 
+  and profile counters. If tail latency violates 10x image unloaded service time at a step, the 
+  function returns the metric before it.
```

Comment:
What happens after the the tail latency is violated? Please clarify this part a bit.
To whom does the function issue the percentage of maximum RPS? And what happens to them after?

Reply:
Done